src/main/java/seedu/address/commons/util/StringUtil.java

```diff
@@ -65,4 +65,16 @@ public static boolean isNonZeroUnsignedInteger(String s) {
             return false;
         }
     }
+
+    /**
+     * Converts a string to title case.
+     *
+     * @param string String input.
+     * @return Title case representation of string.
+     */
+    public static String toTitleCase(String string) {
+        char firstLetter = string.charAt(0);
+        String remaining = string.substring(1);
```

Comment:
Right now it is safe, since i think only @seanjyjy and I are using it. Thanks for the catch though, I will add an assertion there in case someone else happens to use it on a string with length <=1